exercises/react/tests/react.rs

```diff
@@ -206,10 +206,10 @@ fn removing_a_callback_multiple_times_doesnt_interfere_with_other_callbacks() {
     let output = reactor.create_compute(&[input], |v| v[0] + 1).unwrap();
     let callback = reactor.add_callback(output, |v| cb1.callback_called(v)).unwrap();
     assert!(reactor.add_callback(output, |v| cb2.callback_called(v)).is_some());
-    // We want the first remove to be Ok, but we don't care about the others.
+    // We want the first remove to be Ok, but the others should be errors.
```

Comment:
strictly speaking, this line shouldn't be in this PR; it's been wrong ever since I did https://github.com/exercism/rust/pull/191/commits/2602332e4d787dbe0c60eddf48a25c90ab82fce2 as the last commit right before merging https://github.com/exercism/rust/pull/191